src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/task/PaymentMessageProcessTask.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.reform.bulkscan.payment.processor.service.task;
```

Comment:
which packaging convention we choose here? i'm fine but perhaps better to lift it up?